src/openrct2-ui/windows/Footpath.cpp

```diff
@@ -920,7 +920,7 @@ static void window_footpath_place_path_at_point(sint32 x, sint32 y)
     }
 
     // Set path
-    presentType  = default_path_slope[mapElement->properties.path.type & 0x1F];
+    presentType  = default_path_slope[mapElement->properties.path.type & MAP_ELEMENT_SLOPE_MASK];
```

Comment:
You should only use MAP_ELEMENT_SLOPE_MASK if you're reading mapElement->properties.surface.slope.

In this case, you are not, and should therefore create a new constant, namely MAP_ELEMENT_PATH_TYPE_MASK.

Reply:
Done!